### ISSUE TITLE
Make the cross-promotion to modular.im dismissable.

### ISSUE THREAD

**Issue by lampholder** [vector-im/riot-web#9183]
Add a user setting to hide it.

**Comment by aaronraimist** [comment]
Yeah it would be nice if there was a little "X" button to dismiss it.

**Comment by lampholder** [comment]
Yeah, I think this should have a little in-place X button to dismiss it (we should use user settings as a back end to store this dismissal so it doesn't come back; I don't think we should expose this setting in the user settings interface, though).

**Comment by t3chguy** [comment]
This is gone
